src/routes/RequestsRoute.js

```diff
@@ -402,7 +405,9 @@ class RequestsRoute extends React.Component {
       locale: PropTypes.string.isRequired,
     }).isRequired,
     history: PropTypes.object,
-    location: PropTypes.object.isRequired,
+    location: PropTypes.shape({
+      search: PropTypes.string,
+    }).isRequired,
```

Comment:
Thanks for providing this more precise description of the shape of location. `PropTypes.object` borders on useless 🙄 . If you're going to do this, however, the new definition has to be complete: `pathname` is missing and now causes a lint error on line 1033.

Reply:
Thank you @zburke. Fixed